topics/sequence-analysis/tutorials/eDNA-taxonomic-analysis/tutorial.md

```diff
@@ -0,0 +1,324 @@
+> > {: .comment}
+
+{: .hands\_on}
+
+> \<question-title\> Interpreting BLAST Database Choice \</question-title\>
+>
+> 1.  Why is it important to specify the database (like NCBI NT) when performing a BLAST search?
+>
+> > \<solution-title\> Answer \</solution-title\>
+>
+> > 1.  Specifying the database ensures that your sequences are compared against a defined and relevant collection of sequences. Different databases have different content and scope, which can significantly impact the results and the taxonomic assignments you might obtain. Knowing the database also helps in understanding the context and limitations of your search.
+>
+> {: .solution}
+
+{: .question}
+
+# 5. Extracting Taxonomic Information from the BLAST table
+
+## Counting Potential Taxa using **Count**
+
+The BLAST output (in tabular format with extended 25 columns) contains a wealth of information for each aligned sequence. Importantly, **column 25** of this output typically provides the **"subject scientific name"**, which often includes the genus and species identification of the top hit in the database. In this step, we will use the `Count` tool in Galaxy to extract these scientific names and determine how many times each unique name appears in our BLAST results for each of your input files (HP3 R1 and HP3 R2). This will give us a preliminary overview of the potential taxa present in your eDNA samples based on the top BLAST hits.
+
+> \<hands-on-title\> Counting Top Taxonomic Assignments \</hands-on-title\>
+>
+> 1.  Execute the {% tool [Count](Count1) %} with the following parameters:
+>
+> **For HP3 R1 BLAST Results:**
+>
+>   - {% icon param-file %} *"from dataset"*: `output` (output of **NCBI BLAST+ blastn** {% icon tool %} for HP3 R1)
+>   - *"Count occurrences of values in column(s)"*: `c['25']`
+>
+> **For HP3 R2 BLAST Results:**
+>
+>   - Repeat the {% tool [Count](Count1) %} tool with the following parameters:
+>     - {% icon param-file %} *"from dataset"*: `output` (output of **NCBI BLAST+ blastn** {% icon tool %} for HP3 R2)
+>     - {% icon param-file %} *"Count occurrences of values in column(s)"*: `c['25']`
+>
+> > \<comment-title\> Interpreting the Count Output \</comment-title\>
+>
+> > The output of the **Count** tool will be a table showing each unique scientific name found in the 25th column of your BLAST output and the number of times it appeared. Since we set BLAST to only report the top hit for each eDNA sequence, this count represents the frequency of the most likely taxonomic assignment for those sequences.
+> > This output can be used for visualization such as a pie chart.
+> > {: .comment}
+
+{: .hands\_on}
```

Comment:
```suggestion
> > <comment-title> Interpreting the Count Output </comment-title>
> >
> > The output of the **Count** tool will be a table showing each unique scientific name found in the 25th column of your BLAST output and the number of times it appeared. Since we set BLAST to only report the top hit for each eDNA sequence, this count represents the frequency of the most likely taxonomic assignment for those sequences.
> > This output can be used for visualization such as a pie chart.
> > {: .comment}
>
{: .hands\_on}
```